File: SRC - PERL/00 - Estruturas Sequenciais/21.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

#Faça um programa que receba um número real, calcule e mostre:

#a parte inteira desse número;	   #a parte fracionária desse número;	  #o arredondamento desse número


print "Digite um número real > ";
my $numero = <>;
chomp $numero;

my $inteiro = int($numero);
my $fracao = $numero - $inteiro;
my $arredondado = abs($numero);

printf "A parte inteira é %d \n", $inteiro;
printf "A parte fracionária é %f \n", $fracao;
printf "O número arredondado é %d \n", $arredondado;

File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/21.py
#!/usr/bin/python
# coding: latin-1
"""21 - Faça um programa que receba um número real, calcule e mostre:
- a parte inteira desse número;
- a parte fracionária desse número;
- o arredondamento desse número"""

num = float(input("Digite o numero: "))

i = int(num)
f = num - i
a = round(num)

print("A parte inteira desse número: ", i, "\nA parte fracionária desse número: ", f, "\nO arredondamento desse número: ", a)
